01-Authorization-MVC/src/main/java/com/auth0/example/security/SecurityConfig.java

```diff
@@ -1,58 +1,37 @@
 package com.auth0.example.security;
 
 
-import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
-import org.springframework.security.oauth2.core.DelegatingOAuth2TokenValidator;
-import org.springframework.security.oauth2.core.OAuth2TokenValidator;
-import org.springframework.security.oauth2.jwt.*;
 import org.springframework.security.web.SecurityFilterChain;
 
+import static org.springframework.security.config.Customizer.withDefaults;
+
 /**
  * Configures our application with Spring Security to restrict access to our API endpoints.
  */
+@Configuration
 @EnableWebSecurity
```

Comment:
`@EnableWebSecurity` is not needed if it's a Spring Boot app. It's only necessary for Spring apps w/o Boot.